src/amqproxy/cli.cr

```diff
@@ -9,94 +10,27 @@ require "log"
 class AMQProxy::CLI
   Log = ::Log.for(self)
 
-  @listen_address = "localhost"
-  @listen_port = 5673
-  @http_port = 15673
-  @log_level : ::Log::Severity = ::Log::Severity::Info
-  @idle_connection_timeout : Int32 = 5
-  @term_timeout = -1
-  @term_client_close_timeout = 0
+  @config : AMQProxy::Config? = nil
```

Comment:
Possible to make this not optional?

Reply:
Instead of a `run()` method, you can use an initializer and let `cli.new()` set `@config` when initializing, that way I think you can avoid letting it be Nilable.